File: raincoat-fabric/src/main/java/dev/uten2c/raincoat/MixinExtrasInitializer.java
package dev.uten2c.raincoat;

import com.llamalad7.mixinextras.MixinExtrasBootstrap;
import net.fabricmc.loader.api.entrypoint.PreLaunchEntrypoint;

public class MixinExtrasInitializer implements PreLaunchEntrypoint {
    @Override
    public void onPreLaunch() {
        MixinExtrasBootstrap.init();
    }
}

File: raincoat-fabric/src/main/kotlin/dev/uten2c/raincoat/MixinExtrasInitializer.kt
package dev.uten2c.raincoat

import com.llamalad7.mixinextras.MixinExtrasBootstrap
import net.fabricmc.loader.api.entrypoint.PreLaunchEntrypoint

class MixinExtrasInitializer : PreLaunchEntrypoint {
    override fun onPreLaunch() {
        MixinExtrasBootstrap.init()
    }
}
